coalib/settings/Section.py

```diff
@@ -14,6 +14,7 @@ def append_to_sections(sections,
                        key,
                        value,
                        origin,
+                       to_append=False,
```

Comment:
can you put the changes to the setting class to an own commit? Should be simple no?